src/components/task-plan/reading.cjsx

```diff
@@ -116,10 +116,20 @@ ReadingPlan = React.createClass
 
   render: ->
     id = @getParams().id
-    footer = <BS.Button bsStyle="primary" onClick={@publish}>Publish</BS.Button>
+
+    if id
+      plan = TaskPlanStore.get(id)    
```

Comment:
console.log on initial load in render:

getParams().id: undefined
@state.id: _CREATING_0
getParams().id: undefined
@state.id: _CREATING_0

it seems getParams() checks only the location.href?